### ISSUE TITLE
Adding squares onto a project

### ISSUE THREAD

**Issue by SarahSaxifrage** [BiologicalRecordsCentre/plantportal#73]
@andrewvanbreda @sacrevert 
Sorry guys but it's not been fixed, it's still happening.

**Comment by sacrevert** [comment]
@SarahSaxifrage press "Ctrl + shift + R" together and try again, it's probably your browser cache

**Comment by SarahSaxifrage** [comment]
Tried that, but still not working. Plus the guy from the Box Moors Trust is the one who let me know

**Comment by andrewvanbreda** [comment]
Hi @SarahSaxifrage 

Could you let me know what steps you take to reproduce this. I have no problem when I got to Squares Administration on the Meadow Makers project, then I am fully able to use either the Add Square button at the bottom, or Edit Square link on  the rows.

Could you copy the URL address line of the page that isn't work and past it into this thread so I can double check we are talking about the same page.

Thanks

Andy

**Comment by sacrevert** [comment]
@andrewvanbreda I masquaraded as the Box Moor Trust account, and on the Squares administration page ( https://plantportal.ceh.ac.uk/squares-admin-npms-mode?group_id=2167&implicit=f ), the Add square button just takes one back to the Plant Portal homepage

**Comment by andrewvanbreda** [comment]
Hi @sacrevert 
I have corrected this particular project.
The problem is that I have made changes the the structure of the system, so I needed to change all the existing project links manually (the links are normally setup automatically at start of project, so any fixes need manual intervention). 
I made a slight mistake altering links as two pages have similar names "Squares administration" & "Square administration".
I will see if I can rename pages sometime (although that will require further alteration so I won't do it now).
I will have a manual look at the setup of projects to see if I can spot any other mistakes.
Basically the reason why it was working for me is I was looking at a different project.

**Comment by andrewvanbreda** [comment]
Hi @SarahSaxifrage @sacrevert 
That problem also fixed for the following project
"Brampton College: Hendon, Sunnyhill Park, and Brent Reservoir"

Apart from the projects mentioned below, no other projects should have this issue anymore.

There are also some projects that look like they were created before NPMS mode had even finished development so are broken. 
@sacrevert : Can you let me know which of these are in use and I will fix them, otherwise they need deleting, which is now a button that is available on the Edit Project Details page.
Those projects are:

NPMS+
Claypits Monitoring
Saunderton Lee SU8099

Also note that Standard Mode is not finished yet, so the project "Standard mode National Trust test" will not operate correctly.
I have not prevented people creating Standard Mode projects, but if you think this needs restricting let me know.

**Comment by sacrevert** [comment]
>Can you let me know which of these are in use and I will fix them, otherwise they need deleting, which is now a button that is >available on the Edit Project Details page.

This is something for @SarahSaxifrage

**Comment by SarahSaxifrage** [comment]
@andrewvanbreda The projects that most need this element fixing are 
Book Moor Trust Account 
Brampton College
Bristol Plant Monitoring scheme
National trust test
LNDA Habitat condition monitoring
QEOP NPMS
Sherborne Valley Project

Thanks

**Comment by andrewvanbreda** [comment]
Hi @sacrevert @SarahSaxifrage 
I have checked these projects and the configuration looks to be correct (I am not a member of the projects so I can't test by actually going to the problem page in question, but shouldn't be any reason not working)
